bazel/repository_locations.bzl

```diff
@@ -1027,8 +1027,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "f38347360feaaf5b2a733f219c4d8c9660d626f0",
-        sha256 = "bf10de946eb5785813895c2bf16504afc0cd590b9655d9ee52fb1074d0825ea3",
+        version = "ab742cd64f9839e70dd194581e193dadf158928a",
+        sha256 = "0f721d7fd897ed904acfbcfccd43d1a79567084d8234c210e11c370226b0d702",
```

Comment:
Could you update this to the merged commit? Thanks!

Reply:
@PiotrSikora : Do I have to wait for [#209](https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/209) to be merged?

Reply:
No, it doesn't matter, since we're updating the tarball that's updated in #209 in this PR.